Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Call.hs

```diff
@@ -68,18 +77,21 @@ getCustomerMobileNumber callSid callFrom_ callTo_ callStatus = do
         >>= fromMaybeM (RiderDetailsNotFound riderId.getId)
   requestorPhone <- decrypt riderDetails.mobileNumber
   callId <- generateGUID
-  callStatusObj <- buildCallStatus activeRide.id callId callSid (exotelStatusToInterfaceStatus callStatus)
+  callStatusObj <- buildCallStatus activeRide.id callId callSid (exotelStatusToInterfaceStatus callStatus) driver.id digits enteredDtmfFlow
   runTransaction $ QCallStatus.create callStatusObj
   return requestorPhone
   where
     dropFirstZero = T.dropWhile (== '0')
-    buildCallStatus rideId callId exotelCallId exoStatus = do
+    buildCallStatus rideId callId exotelCallId exoStatus driverId alternateNumber enteredDtmfFlow = do
       now <- getCurrentTime
       return $
         SCS.CallStatus
           { id = callId,
             callId = exotelCallId,
             rideId = rideId,
+            driverEntityId = driverId,
+            alternateNumber = alternateNumber,
```

Comment:
Instead of creating two variables. Just use `dtmfNumberUsed` which should have Maybe value. If it is Nothing that mean DTMF flow was not used and if it has some value that means DTMF value was used.